packages/headless/src/controllers/insight/quickview/headless-insight-quickview.ts

```diff
@@ -26,11 +29,21 @@ export function buildQuickview(
   };
 
   const path = '/quickview';
+  const coreProps = {
+    options: {
+      ...props.options,
+      onlyContentURL: true,
```

Comment:
Since `onlyContentURL` is always `true`, I recommend removing the irrelevant properties from the Insight buildQuickview (both the `props` and the return type) to reflect that.

Since that change would be breaking, you may have to simply mark some properties as `@deprecated`, and we can remove them in v3.